test/tools/unified-spec-runner/runner.ts

```diff
@@ -156,10 +156,15 @@ export async function runUnifiedTest(
     const topologyType = ctx.configuration.topologyType;
     if (topologyType === TopologyType.Sharded || topologyType === TopologyType.LoadBalanced) {
       for (const [, collection] of entities.mapOf('collection')) {
-        await utilClient.db(ns(collection.namespace).db).command({
-          distinct: collection.collectionName,
-          key: '_id'
-        });
+        try {
+          // TODO(NODE-4238): create / cleanup entities for each test suite
```

Comment:
Can you provide more context for why this is necessary?  The ticket you have says "Ensuring entities are created on the server prior to starting the tests will remove the need to catch that error." - it would be helpful to elaborate on this - which tests are failing without it and why?

Reply:
Tests are failing due to database not found errors on some server versions.  It appears that the distinct command does not check for database existence on the server, throwing the error - https://jira.mongodb.org/browse/SERVER-60533.

Since other drivers explicitly create the entities on the server, they don't see this issue.

Reply:
Here's an evergreen patch with the try/catch removed that shows the errors - https://spruce.mongodb.com/version/627bc1f2d1fe074fc039b85b/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

It's interesting that